x/migration/recovery/recovery_allowlist.go

```diff
@@ -2114,6 +2120,20 @@ var knownAppStakesAllowlist = []string{
 	"381ADD8CF814F45B8BF6DEA56AC185C2358B8BF2",
 }
 
+var upbitAllowlist = []string{
```

Comment:
this is not only upbit. It's exchanges and DAO and Bridge.

Reply:
How about now?